allofplos/plos_corpus.py

```diff
@@ -374,13 +372,10 @@ def download_amended_articles(directory=corpusdir, tempdir=newarticledir, amende
         amended_article_list = check_for_amended_articles(directory)
     amended_updated_article_list = []
     print("Checking amended articles...")
-    if amended_article_list:
-        for article in tqdm(amended_article_list):
-            updated = download_updated_xml(article)
-            if updated:
-                amended_updated_article_list.append(article)
-    else:
-        pass
+    for article in tqdm(amended_article_list, disable=len(amended_article_list)==0):
```

Comment:
as before, can use "not amended_article_list" instead of "len(amended_article_list)==0"